deploy/templates/bicep/InfrastructureSetup.bicep

```diff
@@ -0,0 +1,209 @@
+param baseName string 
```

Comment:
I'm seeing 3 different naming patterns:
 {basename}{service type abbreviation}
{service type abbreviation}-{basename}
{service type abbreviation}{basename}

can we use just one pattern? since many services don't support using "-" in the name, I think we can get rid of it.